types/leaflet/index.d.ts

```diff
@@ -1408,8 +1408,8 @@ export class Map extends Evented {
     scrollWheelZoom: Handler;
     tap?: Handler;
     touchZoom: Handler;
+    gestureHandling?: Handler;
```

Comment:
GestureHandling is not part of the "original" Leaflet module / scope. It is part of the npm-module [`leaflet-gesture-handling`](https://www.npmjs.com/package/leaflet-gesture-handling).

You should prefer creating an new declaration. Similar to [this one](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/leaflet-fullscreen)...